tests/opentracer/utils.py

```diff
@@ -0,0 +1,39 @@
+from ddtrace.opentracer import Tracer, set_global_tracer
+
+from tests.test_tracer import get_dummy_tracer
```

Comment:
this can stay. We'll remove nose entirely later on during Q3 / Q4 (low priority, but still useful for better testing)